diff -ruN DBD-Sybase-1.15.orig/Makefile.PL DBD-Sybase-1.15/Makefile.PL
--- DBD-Sybase-1.15.orig/Makefile.PL	2012-10-26 21:11:54.000000000 +0200
+++ DBD-Sybase-1.15/Makefile.PL	2017-03-26 13:21:48.539143938 +0200
@@ -139,7 +139,7 @@
 
 	my $libfound = 0;
 	my $libsub   = '';
-  my @libdir = ( 'lib', 'lib64' );
+  my @libdir = ( 'lib', 'lib64', 'libx32' );
   if ($^O =~ /win/i) {
     @libdir = ( 'dll' );
   }
